File: BE/gesture-service/src/main/java/com/hogumiwarts/lumos/gesture/dto/RoutineResponse.java
package com.hogumiwarts.lumos.gesture.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class RoutineResponse {

	private Long routineId;
	private String routineName;
}

File: Android/data/src/main/java/com/hogumiwarts/data/entity/remote/Response/RoutineResponse.kt
package com.hogumiwarts.data.entity.remote.Response

data class RoutineResponse(
    val status: Int,
    val message: String,
    val data: RoutineData
)

data class RoutineData(
    val routineId: Int,
    val routineName: String,
    val routineIcon: String,
    val gestureName: String
)